### PR TITLE
JAX-WS: Refactor of server configuration update

### LINKED ISSUE #26304: JAX-WS: Redesign of server configuration update
Server configuration update was causing timeout issue.

### PR DESCRIPTION
**Pull request by BerksanAtes**
Fixes #26304

### CODE REVIEW

**Comment by BerksanAtes** [comment]
SOE PB on Windows(WIN10_IBM_SEMERUJDK17_CERTIFIED_64) -> https://wasrtc.hursley.ibm.com:9443/jazz/web/projects/WS-CD#action=com.ibm.team.build.viewResult&id=_ou9k0FQNEe6zvJP4IbpA3g

File: dev/com.ibm.ws.jaxws.2.2.webcontainer_fat/fat/src/com/ibm/ws/jaxws/fat/HttpConduitPropertiesTest.java [hunk 1]
```diff
@@ -429,9 +412,10 @@ private String getServletResponseMessage(String servletUrl) throws Exception {
         return result;
     }
 
-    // Waits for config change to be processed after moving the test's ibm-ws-bnd.xml file to the dropins app
-    private void waitForAppRestartAfterConfigChange() throws Exception {
-        // check logs for app restart and servlet readiness
-        server.waitForMultipleStringsInLog(++CONFIG_REFRESH_COUNT, "CWWKG0017I");
+    // Replace server configuration and wait for update to finish
+    private void swapServerConfig(String serverXMLName) throws Exception {
```
Comment (neuwerk) [inline dev/com.ibm.ws.jaxws.2.2.webcontainer_fat/fat/src/com/ibm/ws/jaxws/fat/HttpConduitPropertiesTest.java]: I think the original name stucture of the method is clearer to what it does, it just needs to be updated to reflect what this method does now. I suggest using this naming for the method
```suggestion
    // Replace server configuration and wait for configuration update
    private void waitForServerConfigChange(String serverXMLName) throws Exception {
```

**Review by neuwerk — CHANGES_REQUESTED** [review]
See comment!

File: dev/com.ibm.ws.jaxws.2.2.webcontainer_fat/fat/src/com/ibm/ws/jaxws/fat/HttpConduitPropertiesTest.java (hunk 1 above)
Comment (BerksanAtes) [reply dev/com.ibm.ws.jaxws.2.2.webcontainer_fat/fat/src/com/ibm/ws/jaxws/fat/HttpConduitPropertiesTest.java]: We are not simply waiting for server config change, we are replacing server.xml in the same method as well.

**Review by neuwerk — APPROVED** [review]
LGTM! Thanks Berksan!

**Comment by BerksanAtes** [comment]
#run-libby-bot

**Pull request merged**